src/main/java/li/cil/tis3d/common/module/AudioModule.java

```diff
@@ -53,16 +57,19 @@ public void step() {
 
     @Environment(EnvType.CLIENT)
     @Override
-    public void render(final BlockEntityRenderDispatcher rendererDispatcher, final float partialTicks) {
+    public void render(final BlockEntityRenderDispatcher rendererDispatcher, final float partialTicks,
+                       final MatrixStack matrices, final VertexConsumerProvider vcp,
+                       final int light, final int overlay) {
         if (!getCasing().isEnabled()) {
             return;
         }
 
-        GlStateManager.enableBlend();
+        // XXX Somemthing isn't quite right yet, the sprite seems bigger than in 1.14
```

Comment:
I'll try to provide screenshots as proof that I'm (hopefully) not hallucinating.

Reply:
1.14 first, then my 1.15 version; it's barely noticable at the very edge of the block: https://youtu.be/gMSZ73vMJvQ

Reply:
Weird, I'll have to check that apparently I've been hallucinating, because I could've sworn I didn't see that bleed into the outer "pixel" area, haha.